backup/src/main/java/io/camunda/zeebe/backup/management/BackupService.java

```diff
@@ -104,8 +103,7 @@ public ActorFuture<BackupStatus> getBackupStatus(final long checkpointId) {
 
   @Override
   public ActorFuture<Void> deleteBackup(final long checkpointId) {
-    return CompletableActorFuture.completedExceptionally(
-        new UnsupportedOperationException("Not implemented"));
+    return internalBackupManager.deleteBackup(getBackupId(checkpointId), actor);
```

Comment:
Great that you started working on delete. Before I forget, I have one comment. It should delete backup that is created by another broker as well. So it should iterate over all possible brokerIds. Similar to #10380

Reply:
Good to know :smile:

Reply:
Yes, it should. I can wait until we merge your PR then so I can reuse the same subscriber :+1: